app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,6 @@
+class AdminController < ApplicationController
+  def index
+    @top_customers = Customer.top_customers
+    @pending_invoices = Invoice.pending_invoices
+  end
```

Comment:
you shouldn't need to pass either of these to the view as instance variables, since they're already setup as methods. you can just call `Customer.top_customers.each do` and `Invoice.pending_invoices.each do` in the view directly